specification/dartLangSpec.tex

```diff
@@ -11267,33 +11267,24 @@ \subsection{Function Expressions}
 which is used below and in other sections, as follows:
 
 \begin{itemize}
-\item If $T$ is \code{FutureOr<$S$>} for some $S$ then $\flatten{T} = S$.
+\item If $T$ is a type variable $X$ with bound $B$,
+  or $T$ is a promoted type variable $X & B$,
+  then $\flatten{T} = \flatten{B}$.
```

Comment:
For consistency with other function definitions, we're using an equals sign with a triangle, defined by `\DefEquals{}{}`:
```suggestion
 then \DefEquals{\flatten{$T$}}{\code{\flatten{$B$}}}.
```

Reply:
The other cases don't use `\DefEquals`. I assumed the null safety PRs would make that change, and it was better to *not* change the other lines, so that rebasing got easier.

Reply:
Done.